db/migrate/20161018090517_populate_people_uuid.rb

```diff
@@ -0,0 +1,49 @@
+class PopulatePeopleUuid < ActiveRecord::Migration
+
+  class Person < ActiveRecord::Base
+  end
+
+  def up
+    Person.order(:created_at).pluck(:id).each_slice(1000) { |ids|
+      selects = ids.map { |id|
+        "SELECT '#{id}' AS id, #{create_sql_uuid} AS uuid"
+      }
+
+      sql = "UPDATE people orig
+               JOIN (
+                 #{selects.join("\nUNION ALL\n")}
+               ) joined
+               ON orig.id = joined.id
+               SET orig.uuid = joined.uuid"
+
+      exec_update(sql, "Update people UUIDs", [])
+    }
+  end
+
+  ## UUIDUtils
```

Comment:
Is the point of inlining this code to avoid using code elsewhere that might change and make this migration possibly incorrect later?